.travis.yml

```diff
@@ -0,0 +1,52 @@
+language: go
+
+# Use latest go version
+go:
+- 1.x
+
+# GO111MODULE will force Go modules
+# This will be unnecessary when Go 1.13 lands.
+matrix:
+  include:
+  - os: linux
+    env:
+    - GO111MODULE=on
+
+# Modifies go get flags
+# Can be removed when factom@v1-rollup is merged into master 
+install: true
+
+# Get gox cross-compilation library
+# Move into pegnetMining folder for build
+before_install:
+  - go get github.com/mitchellh/gox
+  - cd pegnetMining
+
+
+# Get specific factom branch
+script: 
+  - go get github.com/FactomProject/factom@v1-rollup
```

Comment:
can this be locked using go.mod?

Reply:
quite sure go.mod being included now makes this call redundant
`go get ...@v1-rollup` creates a go.mod if there isn't one already

Reply:
It should already be locked in the go.mod

Reply:
I think this was done before go mod was in place